docs/root/start/start.rst

```diff
@@ -12,222 +12,12 @@ These examples use the :ref:`v3 Envoy API <envoy_api_reference>`, but use only t
 feature of the API, which is most useful for simple requirements. For more complex requirements
 :ref:`Dynamic Configuration <arch_overview_dynamic_config>` is supported.
 
-.. _install:
-
-Building and installation
--------------------------
-
 .. toctree::
     :maxdepth: 2
 
+    install
+    docker
+    quick-start
+    tools/tools
+    sandboxes/index
```

Comment:
Personally I would swap the order on these?

Reply:
agreed, ill update...

Reply:
i  also swapped docker <> quick-start